weekly/2025/week-18.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Week 18 in Packit
+date: 2025-05-05
+authors: mmassari
+tags:
+  - 2025-April
+  - 2025
+  - April
+---
+
+## Week 18 (April 29th – May 5th)
+
+- Installability check is now being run as part of CI on dist-git pull requests. ([packit-service#2782](https://github.com/packit/packit-service/pull/2782))
+  For more information, please refer to our [documentation on installability.](https://packit.dev/fedora-ci/jobs#installability)
```

Comment:
```suggestion
  For more information, please refer to our [documentation](https://packit.dev/fedora-ci/jobs#installability).
```

Reply:
Actually, I think the convention is to have the implementation PR link in parentheses at the very end, so I would move the _For more info_ line in front of it.